docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -68,6 +68,10 @@ Malware protection also allows you to manage a blocklist to prevent specified ap
 extending the list of processes that {elastic-defend} considers malicious. Use the **Blocklist enabled** toggle
 to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.
 
+If *Detect* malware is enabled, when a malicious file is found it gets quarantined. Specifically {agent} removed from it's current location, obfuscates it (XOR encrypts it with the encryption key `ELASTIC`), moves it to a different folder, and renames it to `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`. {agent} versions 8.5 and later moves quarantined files to the folder `[DriveLetter:]\.quarantine`, unless they are from the `C:` drive, in which case it moves them to `C:\Program Files\Elastic\Endpoint\state\.equarantine`. {agent} versions earlier than 8.5 move quarantined files to `[DriveLetter:]\.quarantine` regardless of which drive they were originally on.
```

Comment:
Shouldn't that be `If *Prevent* malware is enabled` or possibly `If malware is enabled and in *Prevent* mode`...
Because only the Prevent mode would move to the quarantine folder.